zeebe/broker/src/test/java/io/camunda/zeebe/broker/system/partitions/impl/steps/SnapshotDirectorPartitionTransitionStepTest.java

```diff
@@ -89,6 +102,12 @@ void shouldReInstallSnapshotDirector(final Role currentRole, final Role targetRo
     assertThat(transitionContext.getSnapshotDirector())
         .isNotNull()
         .isNotEqualTo(existingSnapshotDirector);
+    if (Role.LEADER.equals(targetRole)) {
+      // verify that the last position is read to notify snapshot director
+      verify(logstreamReader, times(1)).seekToEnd();
+      assertThat(transitionContext.getSnapshotDirector().getCommitPosition())
+          .isEqualTo(LAST_LOG_POSITION);
+    }
```

Comment:
❓ Couldnt we also validate that this snapshot is now valid/confirmed/commited?

Reply:
This test is not taking the snapshot, it is only verifying that commit position is initialized after role transition.